pulpcore/app/apps.py

```diff
@@ -361,3 +367,47 @@ def _populate_artifact_serving_distribution(sender, apps, verbosity, **kwargs):
                     pulp_type="core.artifact",
                     defaults={"base_path": name, "content_guard": content_guard},
                 )
+
+
+def _migrate_remaining_labels(sender, apps, verbosity, **kwargs):
+    try:
+        Label = apps.get_model("core", "label")
+        ContentType = apps.get_model("contenttypes", "ContentType")
+    except LookupError:
+        if verbosity >= 1:
+            print("No labels available.")
+        return
+
+    labeled_ctypes = [
+        ContentType.objects.get(pk=pk)
+        for pk in Label.objects.values_list("content_type", flat=True).distinct()
+    ]
+    for ctype in labeled_ctypes:
+        model = apps.get_model(ctype.app_label, ctype.model)
+
+        if not hasattr(model, "pulp_labels"):
+            if verbosity >= 1:
+                print(
+                    _(
+                        "Warning! "
+                        "Labels for content_type {app_label}.{model} could not be migrated. "
+                        "Model has no labels."
+                    ).format(app_label=ctype.app_label, model=ctype.model)
+                )
+            break
```

Comment:
```suggestion
            continue
```
I think this should be a `continue`, else you will never migrate the other model's pulp_labels even if you could. Also, this probably has to be changed in the other 2 spots. (Copying and pasting code :cry: )

Reply:
Oh, right!